WK2-261: documentation guides improvements after feedback (#14)
wealthkernel/api-documentation-guides: docs/webhooks/Getting-Started.md

## Configuring webhooks through the portal

> Once you’re up and running with ngrok, you’ll need to configure our webhooks. In order to do that, go to our portal and use the left menu to navigate to the webhook pages.

Reviewer: ```suggestion
Once you’re up and running with ngrok you will need to configure your webhooks through our portal, using the left menu to navigate to the webhook pages.
```

Author: Done
